src/player/resources/mod.rs

```diff
@@ -1,7 +1,5 @@
 //! Resources for managing players
 
 mod character;
-mod players;
 
-pub use self::character::{Character, CharacterType, CharactersResource};
-pub use self::players::{PlayerInput, PlayersResource};
+pub use self::character::CharactersResource;
```

Comment:
This module now doesn't have anything. Can we delete it/just use this import elsewhere?